.github/workflows/sync-translations-poeditor.yml

```diff
@@ -0,0 +1,185 @@
+name: Sync Translations with POEditor
+
+on:
+  push:
+    branches:
+      - feat/translation-poeditor-integration
+    paths:
+      - "src/assets/locale/**"
```

Comment:
This workflow will run only if changes are made to files within this specific directory.

Reply:
This should be automated via CI as well:

when I merge a new change on master ...
... `npm run extract-i18n` is automatically executed
... and the resulting latest .xlf files are then sent to POEditor (as is happening here already, I guess)

Reply:
@sleidig If we want `npm run extract-i18n` to execute automatically, we need to set up a separate GitHub Action workflow. This workflow will run whenever there is an update to the master branch. It will:

1. Automatically execute the `npm run extract-i18n` command to update the language files.
2. Commit and push the updated language files to the master branch.
3. Trigger the `Sync Translations with POEditor` workflow.

Is my understanding correct based on your suggestions?

Reply:
Yes, potentially it could run like this. However, this would mean that after almost every commit to master, there will be another (automatic) commit to update the results from "npm run exxtract-i18n" (this is what I pointed out last week during our call). So that would be far from ideal.

Maybe instead the CI could work like this?
- run "npm run extract-i18n within CI without committing it
- check the result of this and trigger the push action to POEditor (if there were changes)
- open a PR with the new xlf files (rather than immediately committing them to master)
- update that branch/PR whenever POEditor users update a translation?

Reply:
Got it. So maybe insteaf of creating the new flow, we can update this workflow. And need to made these changes which I mentioned below.

Reply:
@sleidig Understood. Instead of creating a new flow, I think we can update the existing workflow. Here are the small changes I propose:

1. Remove the condition that restricts the workflow to run only when specific files change.
2. Add a step after the `npm run extract-i18n` command to check if there are any updates on the branch; the workflow should only run if there are updates, otherwise, it should be skipped.
3. Continue sending the updated language files to POEditor and follow the remaining steps as they are already implemented in this workflow.